src/main/java/ro/unibuc/hello/HelloApplication.java

```diff
@@ -72,44 +72,100 @@ public void runAfterObjectCreated() {
 
-		restaurant1 = restaurantRepository.save(new RestaurantEntity(restaurant1.getName(), restaurant1.getEmail(), restaurant1.getAddress(), restaurant1.getOrders()));
+		final OrderEntity finalOrder1 = order1_res1;
+		final OrderEntity finalOrder2 = order2_res1;
+		final OrderEntity finalOrder3 = order3_res1;
+		final OrderEntity finalOrder4 = order4_res1;
+
+
+		ArrayList<OrderEntity> ordersForRestaurant2 = new ArrayList<>(){{
+			add(finalOrder1);
+			add(finalOrder4);
+			add(finalOrder3);
+		}};
 
-		RestaurantEntity restaurant2 = new RestaurantEntity("Maestroooo", "maestrooooo@gmail.com", "Sector 2 Bucuresti", null);
+		ArrayList<OrderEntity> ordersForRestaurant1 = new ArrayList<>(){{
+			add(finalOrder1);
+			add(finalOrder4);
+			add(finalOrder3);
+			add(finalOrder2);
+		}};
 
-		ArrayList<DishesEntity> dishesList4 = new ArrayList<>(Arrays.asList(dish4));
-		OrderEntity order1_res2 = new OrderEntity(client1, restaurant2, dishesList4);
+		ArrayList<OrderEntity> ordersForRestaurant3 = new ArrayList<>(){{
+			add(finalOrder1);
+			add(finalOrder4);
+		}};
 
-		ArrayList<DishesEntity> dishesList5 = new ArrayList<>(Arrays.asList(dish4, dish1));
-		OrderEntity order2_res2 = new OrderEntity(client3, restaurant2, dishesList5);
+		ArrayList<OrderEntity> ordersForRestaurant4 = new ArrayList<>(){{
+			add(finalOrder1);
+			add(finalOrder2);
+			add(finalOrder3);
+		}};
 
-		ArrayList<DishesEntity> dishesList6 = new ArrayList<>(Arrays.asList(dish1, dish2, dish5));
-		OrderEntity order3_res2 = new OrderEntity(client2, restaurant2, dishesList6);
 
-		ArrayList<OrderEntity> ordersForRestaurant2 = new ArrayList<>(Arrays.asList(order1_res2, order2_res2, order3_res2));
 		restaurant2.setOrders(ordersForRestaurant2);
+		restaurant3.setOrders(ordersForRestaurant3);
+		restaurant1.setOrders(ordersForRestaurant1);
+		restaurant4.setOrders(ordersForRestaurant4);
 
-		order1_res2 = orderRepository.save(new OrderEntity(order1_res2.getClient(), order1_res2.getRestaurant(), order1_res2.getDishes()));
-		order2_res2 = orderRepository.save(new OrderEntity(order2_res2.getClient(), order2_res2.getRestaurant(), order2_res2.getDishes()));
-		order3_res2 = orderRepository.save(new OrderEntity(order3_res2.getClient(), order3_res2.getRestaurant(), order3_res2.getDishes()));
 
 		restaurant2 = restaurantRepository.save(new RestaurantEntity(restaurant2.getName(), restaurant2.getEmail(), restaurant2.getAddress(), restaurant2.getOrders()));
+		restaurant3 = restaurantRepository.save(new RestaurantEntity(restaurant3.getName(), restaurant3.getEmail(), restaurant3.getAddress(), restaurant3.getOrders()));
+		restaurant4 = restaurantRepository.save(new RestaurantEntity(restaurant4.getName(), restaurant4.getEmail(), restaurant4.getAddress(), restaurant4.getOrders()));
+		restaurant1 = restaurantRepository.save(new RestaurantEntity(restaurant1.getName(), restaurant1.getEmail(), restaurant1.getAddress(), restaurant1.getOrders()));
```

Comment:
Here we should do something like `restaurant2 = restaurantRepository.save(restaurant2)` in order to not introduce the same restaurant two times in the db.